cdap-examples/pom.xml

```diff
@@ -43,6 +43,7 @@
     <module>UserProfiles</module>
     <module>WebAnalytics</module>
     <module>WordCount</module>
+    <module>WikipediaPipeline</module>
```

Comment:
Is it supposed to be alphabetical? It is currently not. I remember a discussion from a PR last week about adding new apps at the end.